src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -67,7 +67,8 @@ class CpuEngine {
  public:
   static CpuEngine *Get() {
     // I's thread-safe in C++11.
```

Comment:
So we need remove this line of comment?

Reply:
comment was correct, it says mkldnn engine is thread safe to use in mkldnn cpp api.